ui/plugins/eu.esdihumboldt.hale.ui/src/eu/esdihumboldt/hale/ui/io/instance/InstanceExportWizard.java

```diff
@@ -34,6 +36,8 @@
 import eu.esdihumboldt.hale.common.core.io.supplier.FileIOSupplier;
 import eu.esdihumboldt.hale.common.core.io.supplier.Locatable;
 import eu.esdihumboldt.hale.common.core.io.supplier.LocatableInputSupplier;
+import eu.esdihumboldt.hale.common.core.io.supplier.LocatableOutputSupplier;
+import eu.esdihumboldt.hale.common.core.io.supplier.MultiLocationOutputSupplier;
```

Comment:
As the wizard is the only place where the validation is adapted, I assume the validation as part of the headless transformation and possibly the validation via "Transform external data" will not work?
Do you think there is an easy way to apply that there too?